### PR TITLE
JDF-588 Create an Advanced Planner Quickstart

### PR DESCRIPTION
**Pull request by rafabene**

### CODE REVIEW

File: business-resource-planner/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+business-resource-planner: Shows how to solve a vehicle routing problem
+=======================================================================
+Author: Rafael Benevides
```
Comment (rafabene) [inline business-resource-planner/README.md:3]: @ge0ffrey I'm not the Author. I just "adapted" it to JBoss Developer format. Who is the right author of it so I can put his name here. Thanks

Comment (ge0ffrey) [inline business-resource-planner/README.md:3]: It's ok. Doesn't matter. In any case, git history is the only true way to know the real author.
(I wrote most of the vehicle routing example, Frederic Hornain supplied most of the images.)

**Comment by sgilda** [comment]
Rafael, this looks great! 

I plan to make some minor modifications to the README file. I can make them now and issue a pull back to you or I can make them after this is merged. Let me know which you prefer.

**Comment by rafabene** [comment]
@sgilda You can modify youself the Readme file  as I trust you :)

**Comment by sgilda** [comment]
@rafabene : Go ahead and merge then. But please remind me to update the README. :-)

**Comment by ge0ffrey** [comment]
@rafabene Great stuff :)

**Comment by rafabene** [comment]
Thanks @ge0ffrey

**Pull request merged**